### PR TITLE
chore: Customs documents docs

### PR DESCRIPTION
**Pull request by arapicki**
Expand Native Rendering section with customs documents examples.

### CODE REVIEW

File: native-rendering/documents-designer/conditions.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+![Visible shape](./images/condition-shape-visible.png)
+
+By default, the Condition option in the object's properties is empty. This is an option that works on a true/false basis. When we enter a condition here that returns a false value - the given object will not be shown on the preview. 
+
+![Invisible shape](./images/condition-shape-invisible.png)
+
+###### Group of objects
+
+Of course, the use of this function can be more complicated and complex. First of all, we don't have to perform it on a single object, on the entire Group Band. Here, similarly to the above, the simplest example. First the empty condition (default), only this time already for the GROUP, not for a single OBJECT. The next one, however, with the value false.
```
Comment (LukaszParala) [inline native-rendering/documents-designer/conditions.md:19]: Ja bym bardziej sformalizowal:

Function can be more complicated and complex.
We can also perform it on the entire Group Band. Similarly to the above. First the empty condition (default), but this time defined for the GROUP, not for a single OBJECT. The next example is with the false value.

File: native-rendering/documents-designer/conditions.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+![Group of objects](./images/condition-group-of-objects.png)
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/conditions.md:21]: Those images are unreadable. I would split them in two and put one after another.

File: native-rendering/documents-designer/conditions.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+
+###### Possible case
+
+In a more natural example, we want a given Group Band to display if,  country code for sender = "GBR". 
+
+![Ship from country code](./images/ship-from-country-code.png)
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/conditions.md:28]: Those images are unreadable. I would split them in two and put one after another.

File: native-rendering/documents-designer/conditions.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+###### Group of objects
+
+Of course, the use of this function can be more complicated and complex. First of all, we don't have to perform it on a single object, on the entire Group Band. Here, similarly to the above, the simplest example. First the empty condition (default), only this time already for the GROUP, not for a single OBJECT. The next one, however, with the value false.
+
+![Group of objects](./images/condition-group-of-objects.png)
+
+
+###### Possible case
+
+In a more natural example, we want a given Group Band to display if,  country code for sender = "GBR".
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/conditions.md:26]:
```suggestion
In a more natural example, we want a given Group Band to display when sender's country code is "GBR". 
```

File: native-rendering/documents-designer/conditions.md (hunk 1 above)
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/conditions.md:19]:
```suggestion
Conditions usage is not limited to a single object. We can use it with group of objects, called Group Bands, as well. Below are examples of a Group Band with default conditions and when a condition resolved to false.
```
That section seems to be overcomplicated for me.
I would also avoid using words complicated/complex :D Might cause unfavorable associations

Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/conditions.md:19]:
```suggestion
Conditions usage is not limited to a single object. We can use it with group of objects, called Group Bands, as well. Below are examples of a Group Band with default conditions and when a condition resolved to false.
```
That section seems to be overcomplicated for me.
I would also avoid using words complicated/complex :D Might cause unfavorable associations

File: native-rendering/documents-designer/design-window.md [hunk 5]
```diff
@@ -4,12 +4,31 @@ title: Designer Window
...
+
+![Basic operations section](./images/basic-operations-section.png)
+
+Included here are basic operations on the document:
+
+* **New** - create new blank label
+* **Open** - open label from drive
+* **Save** - save current carrier
+* **Save as** - save as .carrier
+* **Preview** - print preview
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/design-window.md:23]:
```suggestion
* **Preview** - display preview
```
So nobody thinks it's printed on some physical printer?

File: native-rendering/documents-designer/design-window.md [hunk 6]
```diff
@@ -19,32 +38,69 @@ title: Designer Window
...
+* **Rotate clockwise** - Rotate object(s) to right
+* **Rotate counterclockwise** - Rotate object(s) to left
+* **Refresh properties on mouse move** - Enabling / disabling refreshing properties while moving objects (with this option is possible to avoid some performance issues)
+* **Alignment strip** - Open additional menu described below
+
+### Alignment section
+
+![Alignment section](./images/alignment-section.png)
+
+Includes all possible alignment options in the document:
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/design-window.md:72]:
```suggestion
Contains all possible alignment options in the document:
```

File: native-rendering/documents-designer/design-window.md [hunk 7]
```diff
@@ -4,12 +4,31 @@ title: Designer Window
...
+* **Save as** - save as .carrier
+* **Preview** - print preview
+* **Settings** - sample input data editor
+* **Save as** - save as archive .zip (all formats)
+
+### New objects section
+
+![New objects section](./images/new-object-section.png)
+
+Included here are actions for adding new objects to the document:
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/design-window.md:31]:
```suggestion
Contains actions for adding new objects to the document:
```

Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/design-window.md:31]:
```suggestion
Contains actions for adding new objects to the document:
```

File: native-rendering/documents-designer/design-window.md [hunk 8]
```diff
@@ -4,12 +4,31 @@ title: Designer Window
...
+## Toolbar
 
-![Toolbar](./images/objects.png)
+![Toolbar](./images/toolbar.png)
 
+### Basic operations section
+
+![Basic operations section](./images/basic-operations-section.png)
+
+Included here are basic operations on the document:
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/design-window.md:17]:
```suggestion
Contains basic operations on the document:
```

File: native-rendering/documents-designer/design-window.md (hunk 6 above)
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/design-window.md:72]:
```suggestion
Contains all possible alignment options in the document:
```

File: native-rendering/documents-designer/design-window.md [hunk 9]
```diff
@@ -19,32 +38,69 @@ title: Designer Window
...
+Includes all possible alignment options in the document:
+
+* **Align Lefts** - align all selected items to left side of first selected item
+* **Align Rights** - align all selected items to right side of first selected item
+* **Align Centers** - align all selected items to center of first selected item
+* **Align Tops** - align all selected items to top of first selected item
+* **Align Bottoms** - align all selected items to bottom of first selected item
+* **Align Middles** - align all selected items to middle of first selected item
+* **Make Same Height** - set height of all selected items to height of first selected item
+* **Make Same Width** - set width of all selected items to height of first selected item
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/design-window.md:81]:
```suggestion
* **Make Same Width** - set width of all selected items to width of first selected item
```
probably

File: native-rendering/documents-designer/design-window.md [hunk 10]
```diff
@@ -19,32 +38,69 @@ title: Designer Window
...
+* **Make Same Size** - make same size of all selected items to height of first selected item
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/design-window.md:82]:
```suggestion
* **Make Same Size** - set size of all selected items to size of first selected item
```
Something like that? At least not "to height", probably

Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/design-window.md:82]:
```suggestion
* **Make Same Size** - set size of all selected items to size of first selected item
```
Something like that? At least not "to height", probably

File: native-rendering/documents-designer/design-window.md [hunk 11]
```diff
@@ -19,32 +38,69 @@ title: Designer Window
...
-| StrToFloat       | _StrToFloat(text)_                              | Converts String to Numeric                                                                                                 |
-| StrToLower       | _StrToLower(text)_                              | Returns text in lower case                                                                                                 |
-| StrToUpper       | _StrToUpper(text)_                              | Returns text in upper case                                                                                                 |
-| StrTrim          | _StrTrim(text)_                                 | Removes all leading and trailing white-space characters from the provided string.                                          |
-| SubStr           | _Substr(text,start,len)_                        | Returns a text fragment                                                                                                    |
+![Properties tab](./images/properties-tab.png)
 
+* **Document** - Properties of the whole document
+* **Group** - Properties of a specific group of objects
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/design-window.md:105]:
```suggestion
* **Group** - Properties of a selected group of objects
```

File: native-rendering/index.md [hunk 12]
```diff
@@ -39,12 +60,52 @@ visual tool to design a document template file.
...
-
 ```bash
 connect env:set RENDERING_HOST={rendering_service_url}
 ```
+
+## Customs Documents
+
+For international shipments, customs declarations CN22 and CN23 are required, as well as an optional commercial invoice.  
+Whether you use a CN22 form or a CN23 depends on the weight and value of the package.  
+Depending on the carrier and the destination you need to include a commercial invoice in addition to the CN22/CN23.
```
Comment (PrzemyslawSpringer) [inline native-rendering/index.md:76]:
```suggestion
Depending on the carrier and the destination, you might need to include a commercial invoice in addition to the CN22/CN23.
```
If I understand the sentence correctly

**Review by pnowicki-auctane — APPROVED** [review]

**Review by ddygas — APPROVED** [review]

**Pull request merged**